staging/src/k8s.io/csi-api/pkg/crd/testdata/csinodeinfo.yaml

```diff
@@ -0,0 +1,37 @@
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  creationTimestamp: null
```

Comment:
Do we even need to specify `creationTimestamp`?

Reply:
it gets serialized by default (the field is not a pointer and is a struct type). See https://github.com/kubernetes/kubernetes/issues/67610 tracking improving this